src/components/atoms/ToggleMapProjectionButton/ToggleMapProjection.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+
+export type MapProjection = 'globe' | 'flat'
+
+type ToggleMapProjectionProps = {
+    currentProjection: MapProjection
+    onClick: () => void
+}
+
+const ToggleMapProjectionButton: React.FC<ToggleMapProjectionProps> = ({ onClick, currentProjection }) => {
```

Comment:
when I click on the back button and then try to click on one of the regions again nothing happens